frontend/lib/src/components/widgets/CustomComponent/ComponentInstance.tsx

```diff
@@ -336,11 +338,12 @@ function ComponentInstance(props: Props): ReactElement {
   // but while we also have not waited until the ready timeout
   const loadingSkeleton = !isReadyRef.current &&
     !isReadyTimeout &&
-    // if height is explicitly set to 0, we don’t want to show the skeleton at all
```

Comment:
nit: the comment is still true, or? I think it makes sense to keep this comment.